src/__tests__/components/permissionPopup.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { shallow, } from 'enzyme'
+import { PERMISSION_QUESTIONS, } from '../../constants'
+import { PermissionPopup, } from '../../components'
+
+describe('check PermissionPopup component', () => {
+  const mockResolve = jest.fn()
+  const mockRefresh = jest.fn()
+  const props = {
+    isWord: false,
+    resolve: mockResolve,
+    refresh: mockRefresh,
+  }
+  const wrapper = shallow(<PermissionPopup {...props} />)
+
+  afterEach(() => {
+    mockRefresh.mockClear()
+    mockResolve.mockClear()
+  })
+
+  test('should render correct permission alert under different props', () => {
```

Comment:
Better split these tests:

`should display PermissionPopup for tag`
`should display PermissionPopup for a word`